### PR TITLE
Book comments

### PR DESCRIPTION
**Pull request by gastonmaspero**
## Summary ##

**Trello card**: https://trello.com/c/TEJO37fv/18-n-62-tabla-de-comentarios

Added comment section

### iPhone X

Simulator Screen Shot - iPhone X - 2019-06-06 at 14 11 09
Simulator Screen Shot - iPhone X - 2019-06-06 at 14 11 05

### CODE REVIEW

File: WBooks/Detail View/BookCommentCell.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import UIKit
+import WolmoCore
+
+class BookCommentCell: UITableViewCell, NibLoadable {
+
+    @IBOutlet weak var profilePic: UIImageView! {
+        didSet {
+            self.profilePic.layer.borderWidth = 1.5
+            self.profilePic.layer.masksToBounds = false
+            self.profilePic.layer.borderColor = UIColor(red: 73/255, green: 194/255, blue: 1, alpha: 1).cgColor
```
Comment (carcos-wolox) [inline WBooks/Detail View/BookCommentCell.swift]: Create a `UIColor` extension, and set all app's colors as computed properties

Comment (gastonmaspero) [reply WBooks/Detail View/BookCommentCell.swift]: ok!

File: WBooks/MainTabBar/LibraryTab.swift [hunk 2]
```diff
@@ -21,6 +21,14 @@ class LibraryTab: UITabBarController {
         return .lightContent
     }
     
+    override open var shouldAutorotate: Bool {
+        return false
+    }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
```
Comment (carcos-wolox) [inline WBooks/MainTabBar/LibraryTab.swift]: If all your app is portrait, you can set it in Target's configuration
image

File: WBooks/Models/Comment.swift [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+struct Comment {
+    
+    var user: User
+    var content: String
+    var book: Book
+    var id: Int
+    
+    var profilePic: UIImage {
+        var image: UIImage = UIImage(named: "no_image_available")!
```
Comment (carcos-wolox) [inline WBooks/Models/Comment.swift]: Also, have an `UIImage` extension for this

File: WBooks/Models/Comment.swift [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+//  Created by Gaston Maspero on 04/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct Comment {
+    
+    var user: User
```
Comment (carcos-wolox) [inline WBooks/Models/Comment.swift]: Does this properties change?

**Review by carcos-wolox — APPROVED** [review]
This is good! I've left some comments.. check them and then you can merge 👍

**Pull request merged**